ci/README.md

```diff
@@ -1,7 +1,9 @@
 # Building and running tests as a developer
 
-CI tests can be run on docker by invoking the script `./ci/run_docker.sh
-./ci/do_ci.sh {TARGET}` where the targets are:
+CI tests can be run inside
+[devcontainer](../CONTRIBUTING.md#devcontainer-setup-for-project)
+by invoking the script
```

Comment:
Could we also add a Dockerfile that creates a container with the current codebase and required dependencies to run the CI tests within that container ?